conv_rnn/data.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import os
```

Comment:
Let's follow PEP8 standard and move the standard library imports before the third party imports.